.github/workflows/ci.yaml

```diff
@@ -36,3 +36,12 @@ jobs:
           # Bazelisk will download bazel to here
           XDG_CACHE_HOME: ~/.cache/bazel-repo
         run: bazel --bazelrc=.github/workflows/ci.bazelrc --bazelrc=.bazelrc test //...
+      - name: bazel test //e2e/...
```

Comment:
bazel test //...

Reply:
actually, this step is only for e2e, so I wanted to emphasise that